components/automate-chef-io/content/docs/eas.md

```diff
@@ -0,0 +1,108 @@
+
+## The Chef Enterprise Application (EAS) Stack
+
+The Chef Enterprise Application Stack (EAS) provides the tools for defining and managing your application throughout its entire lifecycle.
+Chef EAS application, infrastructure, and security automation helps your organization manage the lifecycle of your application with stable, consistent, and repeatable processes, letting you focus on delivering new features--and new value--to your customers.
+Chef EAS automates your application's build and release stages, the configuration for its underlying environments and infrastructure, audits both your application and its infrastructure for security and compliance, while also providing a uniform user experience and visibility accross your entire system.
+
+1. Chef Habitat automates how you build and deploy your applications because delivering applications is where you derive your business value
+1. Chef Infra automates how you configure your infrastructure, because your applications also needs to run on a stable and scalable system
+1. Chef Inspec automates how you validate the security of both your applications and your infrastructure
+1. Chef Automate provides visibility into your applications, their infrastructure, and the compliance status for both.
+1. Chef Workflow provides a unified command-line user experience for all of Chef EAS
+
+Chef Habitat automates your build experience
+
+Chef Automate provides visibility into everything being managed by Chef Habitat, Chef Inspec, and Chef Infra, which are our three automation engines.
+Chef Workstation provides all the tools you'll need to interact with all of the above.
+
+## Chef EAS Application Management with Chef Habitat
+
+Chef Habitat automates building and deploying applications, turning a web of delicate procedures into a resilient and repeatable process.
+A Chef Habitat package contains the compiled binary and a manifest of all the dependencies required for building and running your application.
+This results in a lightweight and portable artifact that you can deploy to bare metal or virtual machines, as well as export to immutable formats, such as Docker containers or Kubernetes pods.
+Bundling applications with a complete dependency manifest ensures that your application behaves as consistently on a developer's laptop as it does in its on-prem VM farm staging environment, and it also behaves as consistently there as it does in its cloud-based deployment environment.
+
+## The Chef Automate EAS Application Tab
+
+The Chef Automate EAS Applications feature allows you to organize and display your applications data from Chef Habitat in an intuitive way.
+The Chef Automate EAS Application Tab provides visibility into your application artifacts and channels by letting you see which versions of your packages are running on your system.
+Chef Automate organizes data from the application and environment tags provided by the the Chef Habitat supervisor.
+
+## Prerequisites
```

Comment:
im not sure the prerequisites parent section is necessary here.. unless yall are planning to add more content to it. and to clarify, def want the glossary, just not sure of this section and sentence saying to go to the glossary